x-pack/plugins/spaces/server/config.ts

```diff
@@ -27,9 +28,19 @@ export function createConfig$(context: PluginInitializerContext) {
 const disabledDeprecation: ConfigDeprecation = (config, fromPath, addDeprecation) => {
   if (config.xpack?.spaces?.enabled === false) {
     addDeprecation({
-      message: `Disabling the Spaces plugin (xpack.spaces.enabled) will not be supported in the next major version (8.0)`,
+      title: i18n.translate('xpack.spaces.deprecations.enabledTitle', {
+        defaultMessage: 'Setting "xpack.spaces.enabled" is deprecated',
+      }),
+      message: i18n.translate('xpack.spaces.deprecations.enabledMessage', {
+        defaultMessage:
+          'Disabling the Spaces plugin (xpack.spaces.enabled) will not be supported in the next major version (8.0).',
+      }),
       correctiveActions: {
-        manualSteps: [`Remove "xpack.spaces.enabled: false" from your Kibana configuration`],
+        manualSteps: [
+          i18n.translate('xpack.spaces.deprecations.enabled.manualStepOneMessage', {
+            defaultMessage: `Remove "xpack.spaces.enabled" from kibana.yml.`,
```

Comment:
Since there is only one step here, can we remove the `1.`?

Reply:
Is there another setting they should use? If yes, we should include that here as a step `2.`. 

Then it would look like:
1. Remove `xpack.spaces.enabled` from kibana.yml.
2. To disable the Spaces plugin, configure `{workaround setting}`.

Reply:
No it's no longer going to be possible to disable the Spaces plugin when we get to 8.0. So there's no other setting to use. I'm not sure if we can remove the `1.` though? @jportner Do you know if it's possible?

Reply:
> I'm not sure if we can remove the `1.` though? @jportner Do you know if it's possible?

It looks like that is rendered by the Upgrade Assistant. 
I'd say it is out of scope for this PR, but we could ask the stack management team to make an enhancement for how this is rendered if there is only one manual step.

Reply:
Created https://github.com/elastic/kibana/issues/112622 to track this.